manager/src/main/java/com/redhat/service/smartevents/manager/models/ServiceLimit.java

```diff
@@ -0,0 +1,48 @@
+package com.redhat.service.smartevents.manager.models;
+
+import java.util.List;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ServiceLimit {
```

Comment:
Perhaps consider a re-name? `ServiceLimitConfig` or something else that makes it clearer this is the representation of a configuration.

Reply:
change it to `ServiceLimitConfig`